src/guiguts/spell.py

```diff
@@ -0,0 +1,326 @@
+"""Search/Replace functionality"""
+
+import importlib.resources
+import logging
+import os
+import regex as re
+
+from guiguts.data import dictionaries
```

Comment:
Yes - that's how it was done in the example of using `importlib.resources`. When we come to actually test packaging/publishing Guiguts, we'll see if I've done it correctly :)

Reply:
Ok, thanks for the explanation. The reason I asked was looking forward to packaging. On Mac, and quite possibly on any platform requiring code signature, the app directory itself can't change (because it invalidates the signature). So when I saw "guiguts.data" I got a bit nervous and wanted to check what that data was. Static dictionaries would be fine. So long as later on we aren't storing anything user-modified there, like global dict or project dicts. But we have the user preferences directory for that now.

Reply:
Thanks for that reminder @tangledhelix. In fact it was your previous work on considering packaging for GG1 that alerted me to that situation, and led to the introduction of the GGprefs directory for user preferences and user "global" dictionaries in GG1. I'm planning to keep the same structure in GG2, at least initially, and everything "under the release" should be treated as read-only.